xbmc/music/Album.cpp

```diff
@@ -176,6 +176,28 @@ std::string CAlbum::GetGenreString() const
   return StringUtils::Join(genre, g_advancedSettings.m_musicItemSeparator);
 }
 
+const std::vector<std::string> CAlbum::GetAlbumArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> albumartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
```

Comment:
I have no idea. This form  of loop is how VECARTISTCREDITS is accessed on all other ocassions, I think it best to be consistent with the rest of the music code don't you?